ted_sws/supra_notice_manager/services/supra_notice_validator.py

```diff
@@ -68,3 +70,33 @@ def summary_validation_for_daily_supra_notice(notice_publication_day: day_type,
     # no notice_ids needed to be stored for supra_notice
```

Comment:
why we keep comments in code?

Reply:
as a reminder of what makes double sense (was requested to store notice_ids in validation report, but they are already in main object). code is commented to be reactivated. will remove it later if no complains will be reported